### PR TITLE
feat(ava): add support for ava 3 out-of-the-box

### PR DESCRIPTION
**Pull request by vinayakkulkarni**
- [x] update configurations to support [Ava 3.0](https://github.com/avajs/ava/releases/tag/v3.0.0) out of the box for a fresh `create-nuxt-app` app.

### CODE REVIEW

**Comment by vinayakkulkarni** [comment]
Found a bug in the configuration from ava 2.x to ava 3.x while performing multiple types of tests (e2e and unit together)

- reference: https://github.com/vinayakkulkarni/nuxt-ava-e2e-unit-testing/pull/98

Fixing it here.

**Comment by vinayakkulkarni** [comment]
Fixed it.

@clarkdo Pl check and review :)

**Comment by vinayakkulkarni** [comment]
Uhh. Any updates?

**Comment by clarkdo** [comment]
@vinayakkulkarni Sorry, I'll do some local tests tonight.

**Comment by vinayakkulkarni** [comment]
> In package.json, there is still a script `test` which will use default ava.config.js.
> 
> As the only difference between e2e.config and unit.config is files, so how do you think we use `ava.config.js` for both of them and set `--match='...'` in script `test:unit` and `test:e2e` ?

Hmm.. lemme test it.

**Comment by vinayakkulkarni** [comment]
--match matches the file names.

PS. Ava also accepts [glob patterns](https://github.com/avajs/ava/blob/master/docs/05-command-line.md#cli) in parameter, I've updated it [here](https://github.com/vinayakkulkarni/nuxt-ava-e2e-unit-testing/pull/106/commits/05bb9e0f0eef385a779dc4e3e4b1c1e873017b5e)

File: test/snapshots/index.test.js.md [hunk 1]
```diff
@@ -2186,8 +2185,8 @@ Generated by [AVA](https://avajs.dev).
         generate: 'nuxt generate',
         start: 'nuxt start',
         test: 'ava',
-        'test:e2e': 'cross-env TEST=e2e ava --config e2e.config.js',
-        'test:unit': 'cross-env TEST=unit ava --config unit.config.js',
+        'test:e2e': 'cross-env TEST=e2e ava --config ava.config.cjs ./test/e2e/**/*',
```
Comment (clarkdo) [inline test/snapshots/index.test.js.md]: Can `--config ava.config.cjs` be ignored ?

Comment (vinayakkulkarni) [reply test/snapshots/index.test.js.md]: Yep. As it'll automatically find a ava.config.* file in the root dir.

I'll update this.

**Comment by vinayakkulkarni** [comment]
@clarkdo : can you please review/test?

**Comment by clarkdo** [comment]
Thanks！

**Pull request merged**